DiyorMarket.MVC/Lesson11/Views/Customers/Index.cshtml

```diff
@@ -5,18 +5,29 @@
     Layout = "~/Views/Shared/_DataGridLayout.cshtml";
 }
 
-<head>
-    <link rel="stylesheet" href="~/css/animation.css" />
-</head>
-
 <!-- Navigation link -->
-<nav aria-label="breadcrumb" style="margin-left: 20px;">
+<nav aria-label="breadcrumb">
     <ol class="breadcrumb">
         <li class="breadcrumb-item"><a href="/dashboard/index"><i class="fa fa-home" style="color: white"></i></a></li>
         <li class="breadcrumb-item active" aria-current="page">Customers List</li>
```

Comment:
It's better to name it **Customers** instead of **Customers List**. List does not give any valuable addition.